### PR TITLE
Added vuejs-compose example

### PR DESCRIPTION
**Pull request by bigpod98**

### CODE REVIEW

File: vuejs-compose/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version: '2'
+services:
+  web:
+    build: vuejs
+    image: vuejs_web
+    ports:
+      - 80:80
+    volumes:
```
Comment (kcq) [inline vuejs-compose/docker-compose.yml:8]: should delete these volumes (not needed here anymore)

File: vuejs-compose/docker-compose.slim.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+version: '2'
+services:
+  web: 
+    image: vuejs_web:latest
```
Comment (kcq) [inline vuejs-compose/docker-compose.slim.yml:4]: it should reference vuejs_web.slim

File: vuejs-compose/docker-compose.slim.yml [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+    ports: 
+      - '1300:1300'
```
Comment (kcq) [inline vuejs-compose/docker-compose.slim.yml:6]: is it really listening on port 1300?

File: vuejs-compose/docker-compose.slim.yml (hunk 2 above)
Comment (bigpod98) [reply vuejs-compose/docker-compose.slim.yml:4]: all of them good catches

File: vuejs-compose/vuejs/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,11 @@
+FROM node:lts-bullseye as build-stage
+WORKDIR /app
+COPY package*.json ./
+RUN npm install
+COPY ./ .
+RUN npm run build
+
+FROM nginx as production-stage
```
Comment (kcq) [inline vuejs-compose/vuejs/Dockerfile:8]: Combining nginx and the vue app is a pretty nice example. We also need a standalone (and preferrably single stage) example where it's just the vue app (let's make it a separate demo app)

**Comment by kcq** [comment]
thanks for the PR @bigpod98 !

**Pull request merged**